python/mxnet/contrib/onnx/_import/op_translations.py

```diff
@@ -417,6 +458,16 @@ def flatten(attrs, inputs, proto_obj):
     new_attrs = translation_utils._remove_attributes(attrs, ['axis'])
     return 'Flatten', new_attrs, inputs
 
+def clip(attrs, inputs, proto_obj):
+    """Clips (limits) the values in an array."""
+    new_attrs = translation_utils._fix_attribute_names(attrs, {'min' : 'a_min',
+                                                               'max' : 'a_max'})
+    if 'a_max' not in new_attrs:
+        new_attrs = translation_utils._add_extra_attributes(new_attrs, {'a_max' : np.inf})
+    if 'a_min' not in new_attrs:
+        new_attrs = translation_utils._add_extra_attributes(new_attrs, {'a_min' : -np.inf})
```

Comment:
can we use np.ninf? I think that is float, please check once.

Reply:
np.inf can be used as float - https://stackoverflow.com/questions/42315541/difference-between-np-inf-and-floatinf